src/components/externalLinks/ExternalLinksCRUD/ConfigureLinkAction.tsx

```diff
@@ -0,0 +1,185 @@
+            default:
+                return <></>
+        }
+    }
+
+    return (
+        <div id={`link-action-${index}`} className="configure-link-action-wrapper">
+            <div className="link-monitoring-tool mb-8">
+                <div className="monitoring-tool mr-8">
+                    <span>Monitoring Tool*</span>
+                    <ReactSelect
+                        placeholder="Select tool"
+                        name={`monitoring-tool-${index}`}
+                        value={link.tool}
+                        options={monitoringTools}
+                        isMulti={false}
+                        hideSelectedOptions={false}
+                        onChange={(selected) => onMonitoringToolSelection(index, selected)}
+                        components={{
+                            IndicatorSeparator: null,
+                            ClearIndicator: null,
+                            Option: customOptionWithIcon,
+                            ValueContainer: customValueContainerWithIcon,
+                        }}
+                        styles={{
+                            ...multiSelectStyles,
+                            ...customMultiSelectStyles,
+                            option: (base, state) => ({
+                                ...customMultiSelectStyles.option(base, state),
+                                backgroundColor: state.isSelected
+                                    ? 'var(--B100)'
+                                    : state.isFocused
+                                    ? 'var(--N100)'
+                                    : 'white',
+                                color: state.isSelected ? 'var(--B500)' : 'var(--N900)',
+                            }),
+                            container: (base, state) => ({
+                                ...base,
+                                marginTop: '6px',
+                            }),
+                            control: (base, state) => ({
+                                ...customMultiSelectStyles.control(base, state),
+                                width: '150px',
+                            }),
+                            valueContainer: (base) => ({
+                                ...base,
+                                padding: '0 10px',
+                            }),
+                            placeholder: (base) => ({
+                                ...base,
+                                color: 'var(--N500)',
+                            }),
+                        }}
+                    />
+                    {link.invalidTool && getErrorLabel('tool')}
+                </div>
+                <div className="link-name mr-12">
+                    <label>Name*</label>
+                    <input
+                        placeholder="Enter name"
+                        value={link.name}
+                        onChange={(e) => onNameChange(index, e.target.value)}
```

Comment:
we can avoid inline function using data-atributes

Reply:
will be covered in the next PR.